src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistDetailsList.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box } from "@artsy/palette"
+import React from "react"
+import { useSystemContext } from "v2/Artsy"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PartnerArtistDetailsFragmentContainer } from "../../Components/PartnerArtists"
+import { PartnerArtistDetailsList_edges } from "v2/__generated__/PartnerArtistDetailsList_edges.graphql"
+import { PartnerArtistDetailsListQuery } from "v2/__generated__/PartnerArtistDetailsListQuery.graphql"
+
+export interface PartnerArtistDetailsListProps {
+  edges: PartnerArtistDetailsList_edges
+}
+
+// TODO: Rewrite to using infinite scroll
+export const PartnerArtistDetailsList: React.FC<PartnerArtistDetailsListProps> = ({
+  edges,
+}) => {
+  return (
+    <Box mt={4}>
+      {edges
+        .filter(artist => artist.isDisplayOnPartnerProfile)
+        .filter(artists => artists.counts.artworks > 0)
+        .map(edge => {
+          return (
+            <PartnerArtistDetailsFragmentContainer
+              key={edge.node.id}
+              artist={edge.node}
+            />
+          )
+        })}
+    </Box>
+  )
+}
+
+export const PartnerArtistDetailsListFragmentContainer = createFragmentContainer(
+  PartnerArtistDetailsList,
+  {
+    edges: graphql`
+      fragment PartnerArtistDetailsList_edges on ArtistPartnerEdge
+        @relay(plural: true) {
+        isDisplayOnPartnerProfile
+        representedBy
+        counts {
+          artworks
+        }
+        node {
+          id
+          ...PartnerArtistDetails_artist
+        }
+      }
+    `,
+  }
+)
+
+export const PartnerArtistDetailsListRenderer: React.FC<{
+  partnerId: string
+}> = ({ partnerId, ...rest }) => {
+  const { relayEnvironment } = useSystemContext()
+
+  return (
+    <QueryRenderer<PartnerArtistDetailsListQuery>
+      environment={relayEnvironment}
+      query={graphql`
+        query PartnerArtistDetailsListQuery($partnerId: String!) {
+          partner(id: $partnerId) @principalField {
+            artists: artistsConnection(first: 10) {
+              edges {
+                ...PartnerArtistDetailsList_edges
```

Comment:
Also, I'm getting an error when trying to load the artist's biography when using artistsConnection. it looks like metaphysics can't resolve this field.

![image](https://user-images.githubusercontent.com/79979820/114036813-0e224300-9889-11eb-867c-8e0eb3008f7f.png)

Do you have any idea how to fix it?

Reply:
Would just have to debug the resolver for that field; something is null when trying to convert the Markdown

Reply:
So, after some research, I found that [artistsConnection](https://github.com/artsy/metaphysics/blob/a1d8173d7b788dd18ec8578524cb5c9a4118ef66/src/schema/v2/partner.ts#L122) definitely can't resolve `biography` field.
The response from the REST API(`/partner/${id}/partner_artists`) doesn't contain any information about the artist's biography: 
![image](https://user-images.githubusercontent.com/79979820/114194929-a6d3c400-9958-11eb-877b-df4f4e75a7b0.png)

@sweir27 @dzucconi Do you have any idea?

Reply:
Oh interesting-- Might be using a `list` endpoint, which doesn't include the full JSON response for the object.

This can be fixed, but it might be worth pulling it out of this PR so it's not blocking this one.

Reply:
@sweir27 Is it the metaphysics or gravity problem(I mean REST API, not sure that it's the right name of the project)? I mean can it be fixed by our team or we should create a ticket for someone?

Reply:
Let's create a ticket on the PX board-- I can do some investigation to see whether it will be a change in metaphysics/force or in gravity and let you know.